newIDE/app/src/UI/Text.js

```diff
@@ -11,7 +11,7 @@ type textSize =
   | 'body'
   | 'body2';
 
-type textColor = 'error' | 'primary' | 'secondary';
+type textColor = 'error' | 'primary' | 'secondary' | 'inherit';
```

Comment:
This is a new case where we want the color to be the same on all themes.
Our Text component does not allow this for the moment.
I think setting the color on the container, and using inherit is an OK practice, rather than allowing the `<Text/>` to have a color that can be overriden

Reply:
I think in some exceptional use cases where it's the container that defines the color, it's ok to say "this text has no color, it's inheriting it from the parent".